### PR TITLE
add signalfx terraform modules

### PR DESCRIPTION
**Pull request by xp-1000**

### CODE REVIEW

File: tf-modules-signalfx.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## terraform-signalfx-detectors
+[![github](https://img.shields.io/badge/source-github-black.svg?style=flat-square&logo=github)](https://github.com/claranet/terraform-signalfx-detectors) [![registry](https://img.shields.io/badge/terraform-registry-623CE4.svg?style=flat-square&logo=terraform)](https://registry.terraform.io/modules/claranet/detectors/signalfx)
+
+Terraform list of modules for signalfx detectors
```
Comment (pdecat) [inline tf-modules-signalfx.md]: What about this?
```suggestion
Terraform modules for SignalFx detectors
```

Comment (pdecat) [reply tf-modules-signalfx.md]: Or
```suggestion
List of terraform modules for SignalFx detectors
```

Comment (xp-1000) [reply tf-modules-signalfx.md]: done

**Review by pdecat — APPROVED** [review]
LGTM!

**Review by rossifumax — APPROVED** [review]
LGTM ! :+1:

**Pull request merged**